packages/utils/src/index.js

```diff
@@ -1,7 +1,35 @@
 // @flow
 import type { RegisteredCache, EmotionCache, SerializedStyles } from './types'
 
-let isBrowser = typeof document !== 'undefined'
+const isBrowser = typeof document !== 'undefined'
+
+const getGlobalVersions = (context: any) => {
+  const globalVersionsKey = '__EMOTION_VERSIONS__'
+  if (!context[globalVersionsKey]) {
+    context[globalVersionsKey] = {}
+  }
+  return context[globalVersionsKey]
+}
+
+type ModuleVersions = { [key: string]: string }
+
+const versions: ModuleVersions = getGlobalVersions(isBrowser ? window : global)
+
+export function registerEmotionModule(name: string, version: string) {
+  if (versions[name] && versions[name] !== version) {
+    console.warn(
+      `Package ${name}@${
+        versions[name]
+      } is already registered. Running ${version} may cause problems.`
+    )
+  }
+  versions[name] = version
+}
+
+registerEmotionModule(
+  require('../package.json').name,
+  require('../package.json').version
+)
```

Comment:
This is more here for usage, it may be decided that it's OK for this module to have multiple concurrent versions.